src/deepsparse/loggers/helpers.py

```diff
@@ -250,10 +253,12 @@ def check_identifier_match(
         pattern = template[3:]
         return re.match(pattern, identifier) is not None, None
     if template.startswith("category:") and category is not None:
-        template_category = template[9:]
+        template_identifier = template[9:]
+        template_category, *remainder = template_identifier.split("/")
         template_category = MetricCategories(template_category)  # parse into Enum
         category = MetricCategories(category)  # ensure in Enum form
-        return template_category == category, None
+        remainder = None if not remainder else "/".join(remainder)
```

Comment:
I feel like the structure of those identifiers is quickly getting out of hand. For 1.4 I feel we are in a good state, but before we start introducing complexity for the next release, I would focus on a concrete vision of how the identifiers should look like, what are the delimiters, what is their meaning etc.

Reply:
yes, this sounds good. agreed on getting this simplified